.github/workflows/trivy-release.yml

```
@@ -0,0 +1,116 @@
+        required: true
+    
+    inputs:  
+      image_repo:
+        description: 'Base image repository name'
+        required: true
+        type: string
+      digest_artifact_name:
+        description: 'Digest artifact suffix; defaults to image_repo when omitted'
+        required: true
+        type: string
+        default: ''
```

actionlint (1.7.12)

[error] input "digest_artifact_name" of workflow_call event has the default value "", but it is also required. if an input is marked as required, its default value will never be used

(events)